source/install/cloud-changelog.md

```diff
@@ -3,12 +3,45 @@
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
 Latest Mattermost Cloud releases:
+- [Release 2021-11-10](#release-2021-11-10)
 - [Release 2021-10-27](#release-2021-10-27)
 - [Release 2021-10-13](#release-2021-10-13)
 - [Release 2021-09-29](#release-2021-09-29)
 - [Release 2021-08-12](#release-2021-08-12)
 - [Release 2021-07-29](#release-2021-07-29)
-- [Release 2021-07-15](#release-2021-07-15)
+
+## Release 2021-11-10
+
+### Improvements
+
+#### User Interface (UI)
+ - Added online status to profile images on user autocomplete.
+ - App Commands now have an option to open them as modals.
+ - Added support for navigating through Collapsed Reply Threads via arrow keys.
+ - Added support for focusing the input box in Collapsed Reply Threads while typing.
+ - Added support for blurring the input box in Collapsed Reply Threads on pressing Escape.
+ - Adjusted the channel override desktop notification preference for Threads.
+ - Added a **Click to open thread** setting for all users.
+ - Added lang/tz defaults to Luxon.
```

Comment:
What does this mean?

Reply:
Is this a reference to supporting Swahili in product translations?

Reply:
@calebroseland may have more details.

Reply:
Sorry, I didn’t think to check country codes before using a short form! Looking at the other release notes—this one could be omitted, as there isn’t a direct change in UX. 

Expanded: “Added language and timezone defaults to Luxon for use in time formatting.”

Reply:
It seems it's not necessarily needed here, so I've removed it.